klayout/drc/rule_decks/drc_bjt.drc

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
```

Comment:
should we check this from the caller side instead?

Reply:
@proppy No, I prefer to make all the unique checks related to the table here to make the include process easier to handle. There will be a lot of if conditions in the main file with names and stuff like that.

Reply:
sorry for the confusion, here I'm just referring to the `if FEOL` check.

The reason I'm suggesting to move the conditional to the main drc script, is that the `FEOL` variable being checked is also defined there. 

My impression was that it would allow to decouple the DRC rules from the context where it's being included and allow it to run independently, but are there more dependencies between the 2 scripts?

Reply:
#58